tests/robustness/failpoints.go

```diff
@@ -527,3 +529,48 @@ func (f delayPeerNetworkFailpoint) Name() string {
 func (f delayPeerNetworkFailpoint) Available(config e2e.EtcdProcessClusterConfig, clus e2e.EtcdProcess) bool {
 	return config.ClusterSize > 1 && clus.PeerProxy() != nil
 }
+
+type killAndGofailSleep struct {
+	failpoint string
+	time      time.Duration
+}
+
+func (f killAndGofailSleep) Inject(ctx context.Context, t *testing.T, lg *zap.Logger, clus *e2e.EtcdProcessCluster) error {
+	member := clus.Procs[rand.Int()%len(clus.Procs)]
+	for member.IsRunning() {
+		err := member.Kill()
+		if err != nil {
+			lg.Info("Sending kill signal failed", zap.Error(err))
+		}
+		err = member.Wait(ctx)
+		if err != nil && !strings.Contains(err.Error(), "unexpected exit code") {
```

Comment:
The `Wait` seems to return `ctx.Err` timeout error only.

Reply:
Copied from killFailpoint. Handling "unexpected exit code" comes from https://github.com/etcd-io/etcd/pull/14672